src/App.tsx

```diff
@@ -14,11 +14,96 @@ const goodsFromServer: string[] = [
   'Garlic',
 ];
 
-const App: React.FC = () => (
-  <div className="App">
-    <h1>Goods</h1>
-    {goodsFromServer.length}
-  </div>
-);
+export const App: React.FC = () => {
+  const [goods, setGoods] = useState(goodsFromServer);
+  const [isVisible, setVisibility] = useState(false);
 
-export default App;
+  const showGoods = () => {
+    setVisibility(!isVisible);
+  };
```

Comment:
function has name showGoods, to show tou shoul set true. If it's toggle give function right name and use functional set state 
```suggestion
  const showGoods = () => {
    setVisibility(true);
  };
```